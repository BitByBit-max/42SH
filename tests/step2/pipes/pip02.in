ls  -al / | grep "^d" | awk '{print $9}' | sort -r | head -n 5
