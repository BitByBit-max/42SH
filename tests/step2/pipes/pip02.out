var
usr
tmp
sys
srv
